src/org/opendatakit/briefcase/util/ServerUploader.java

```diff
@@ -204,7 +212,14 @@ public boolean uploadFormAndSubmissionFiles(List<FormStatus> formsToTransfer) {
       File briefcaseFormMediaDir = FileSystemUtils.getMediaDirectoryIfExists(briefcaseLfd.getFormDirectory());
 
       boolean outcome;
-      outcome = uploadForm(formToTransfer, briefcaseFormDefFile, briefcaseFormMediaDir);
+      boolean existsAlready = http.execute(server.containsFormQuery(formToTransfer.getFormDefinition().getFormId()));
```

Comment:
If you inline `existsAlready` or make it a function (lazy val in Scala), you can avoid the RPC when `forceSendBlank` is true, because the `if` will short circuit.

Reply:
It didn't occur to me making it lazy. Thanks for the suggestion!